txircd/modules/extra/ratelimit.py

```diff
@@ -11,6 +12,16 @@ class RateLimit(ModuleData):
 	def actions(self):
 		return [("commandpermission", 100, self.recvCommand)]
 
+	def verifyConfig(self, config):
+		if "ratelimit" in config:
+			if not isinstance(config["ratelimit"], dict):
+				raise ConfigValidationError("ratelimit", "value must be a dictionary")
+			for key, value in config["ratelimit"].itemitems():
+				if key not in ["limit", "kill_limit", "interval"]:
```

Comment:
If you use parentheses instead, it'll make it a tuple.
I don't know if there's a difference that matters for this purpose, but tuples are what we use everywhere else we do this.